css/style.css

```diff
@@ -513,6 +515,104 @@ ul {
   font-size: 15px;
 }
 
+/* POPUP WINDOW */
+.modal {
+  background: rgba(75, 88, 105, 0.25);
+  backdrop-filter: blur(2px);
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 200;
+  display: grid;
+  place-items: center;
+  overflow: auto;
+  padding: 1em;
+}
+
+.modal__card {
+  background-color: var(--CL-neutral-white);
+  border-radius: 16px;
+  padding: 1em;
+  width: min(100%, 1156px);
+}
+
+.modal__title {
+  font-size: 30px;
+  color: var(--CL-primary-dark);
+}
+
+.modal__content {
+  gap: 1em;
+}
+
+.modal__profile {
+  font-size: 13px;
+  gap: 2em;
+}
+
+.modal__list {
+  padding: 0;
+  list-style-type: disc;
+  gap: 1.8em;
+}
+
+.modal__item {
+  color: var(--CL-primary-gray-300);
+  font-weight: 600;
+}
+
+.modal__image {
+  width: 100%;
+  height: 220px;
+  object-fit: cover;
+  border-radius: 8px;
+}
+
+.modal__system {
+  gap: 0.75em;
+  padding: 0.25em 0;
+}
+
+.modal__description {
+  font-size: 15px;
+  color: var(--CL-primary-gray-650);
+}
+
+.modal__tags {
+  gap: 0.5em;
+  flex-wrap: wrap;
+}
+
+.modal__tag {
+  border-radius: 8px;
+  font-size: 12px;
+  color: var(--CL-primary-blue);
+  padding: 4px 12px;
+  background-color: var(--CL-primary-light);
+}
+
+.modal__actions {
+  gap: 0.5em;
+}
+
+.modal__button {
+  flex: 1;
+  background: none;
+  font-size: 15px;
+  gap: 1em;
+}
```

Comment:
- [ ] On the desktop version, your button does not match the given design as shown below. Please ensure that the button designs match.

### Your design
![Screenshot 2023-06-21 at 20 39 49 (2)](https://github.com/Rysth/Portfolio-Project/assets/64774936/5c4d8100-254f-4708-b22f-9f0a4bac63ff)

### Figma design
![Screenshot 2023-06-21 at 20 39 39 (2)](https://github.com/Rysth/Portfolio-Project/assets/64774936/35ec8d26-5610-46b1-ab79-5c12205c1e74)

Reply:
Thanks, @Ghiftee 

### 🏅 Fixeds Implemented

List of things that were fixed based on the last review:

- ✅ Fixed the `buttons` design from the PopUp Window.

_John Palacios_
_Microverse Student_